src/settings/FinePolicy/FinePolicyDetail.js

```diff
@@ -71,10 +72,13 @@ class FinePolicyDetail extends React.Component {
 
   render() {
     const {
-      initialValues: policy = {},
+      initialValues: policy,
       stripes: {
         connect,
-      }
+      },
+      intl: {
+        formatMessage,
```

Comment:
Could you please check posibility of `formatMessage` using in all cases and drop `FormattedMessage`?

Reply:
I'm not sure we need to do it. We use `formatMessage` to be able to test some cases. So if we can test without `formatMessage` I would leave `FormattedMessage` component. Let us discuss

Reply:
Ok, no problem, your point of view make sense.